Automattic/wp-calypso
# Update/2fa sms copy

This pull request fixes #14308 by updating the copy used for the SMS verification form and for the noticed displayed to confirm an SMS has been sent.
  
#### Testing instructions
  
1. Run `git checkout update/2fa-sms-copy` and start your server.
2. Visit `/log-in` and enter the credentials for an account with 2FA enabled.
3. If necessary, click the "Code via text message" option.
4. The form should say: Enter the code from the text message we sent you.
5. The success notice should say: Message sent to phone number ending in 8888.
  
#### Reviews
